bin/index.js

```diff
@@ -45,6 +45,16 @@ config(function (versions) {
   }
 
   async.doWhilst(getBin, whileCondition, function () {
-    if (switcherBin) return switcher(switcherBin)
+    if (switcherBin) {
+      var switcherProcess = switcher(switcherBin)
+
+      switcherProcess.on('close', function (code) {
+        if (code === 0) {
+          console.log(maxSatisfyVersion)
```

Comment:
The point is, it's a very tricky solution for nvm that affects n users. 

I understand that you need the echo for setup correctly nvm, but, as n user, I don't need it.
one question, are you considered move from nvm to n?

As you can see, a lot of problems related with nvm.